Index: freetype-2.3.11/src/pfr/pfrsbit.c
===================================================================
--- freetype-2.3.11.orig/src/pfr/pfrsbit.c
+++ freetype-2.3.11/src/pfr/pfrsbit.c
@@ -600,8 +600,8 @@
 
     /* get the bitmap metrics */
     {
-      FT_Long   xpos, ypos, advance;
-      FT_UInt   xsize, ysize, format;
+      FT_Long   xpos = 0, ypos = 0, advance;
+      FT_UInt   xsize = 0, ysize = 0, format = 0;
       FT_Byte*  p;
 
 
